components/item/SummaryInfo.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="card mb-3 ">
+    <header
+      v-if="titles"
+      class="card-heading px-4 pt-4"
+    >
+      <template
+        v-for="(heading, index) in titles"
+      >
+        <h1
```

Comment:
The main title in the design has a font-weight of normal, here it is bold (600). I haven't found any styling for subtitles in Zeplin. 

The title is also slightly smaller in the design, 28px.